src/api/Item.js

```diff
@@ -200,7 +201,12 @@ class Item extends Base {
             .delete({ url })
             .then(this.deleteSuccessHandler)
             .catch((e: $AxiosError<any>) => {
-                this.errorHandler(e);
+                const { response } = e;
+                if (response?.status === HTTP_STATUS_CODE_NOT_FOUND) {
+                    this.deleteSuccessHandler();
```

Comment:
this seems odd to do semantically since it goes through the success flow on an error. what about passing in an `errorCallback` to `deleteItem` that refreshes the collection?
https://github.com/box/box-ui-elements/blob/6aef87b8ca7316e871abaa0584abf6e4c918e65c/src/elements/content-explorer/ContentExplorer.js#L1088-L1091

```
this.api.getAPI(type).deleteItem(
    selected,
    () => {
        onDelete(cloneDeep([selected]));
        this.refreshCollection();
    },
    () => {
        this.refreshCollection();
    },
);
```

Reply:
thats what I ended up doing after reading your comment